From avorontsov@ru.mvista.com  Thu Aug 20 14:35:30 2009
From: Anton Vorontsov <avorontsov@ru.mvista.com>
Date: Wed, 19 Aug 2009 02:23:35 +0400
Subject: USB: fsl_qe_udc: Add fsl,mpc8323-qe-usb compatible entry
To: Greg Kroah-Hartman <gregkh@suse.de>, David Brownell <dbrownell@users.sourceforge.net>
Cc: Li Yang <leoli@freescale.com>, Scott Wood <scottwood@freescale.com>, Kumar Gala <galak@kernel.crashing.org>, linux-usb@vger.kernel.org, linuxppc-dev@ozlabs.org
Message-ID: <20090818222335.GA28135@oksana.dev.rtsoft.ru>
Content-Disposition: inline


Current bindings specify that "fsl,mpc8323-qe-usb" compatible entry
should be used as a base match for QE UDCs, so update the driver to
comply with the bindings.

Signed-off-by: Anton Vorontsov <avorontsov@ru.mvista.com>
Cc: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/fsl_qe_udc.c |    4 ++++
 1 file changed, 4 insertions(+)

--- a/drivers/usb/gadget/fsl_qe_udc.c
+++ b/drivers/usb/gadget/fsl_qe_udc.c
@@ -2751,6 +2751,10 @@ static int __devexit qe_udc_remove(struc
 /*-------------------------------------------------------------------------*/
 static struct of_device_id __devinitdata qe_udc_match[] = {
 	{
+		.compatible = "fsl,mpc8323-qe-usb",
+		.data = (void *)PORT_QE,
+	},
+	{
 		.compatible = "fsl,mpc8360-qe-usb",
 		.data = (void *)PORT_QE,
 	},
